AEFeinstein/Super-2024-Swadge-FW
# Gems and rings

### Description

Pretty much the whole game

### Test Instructions

* Play a game against someone else
* Unlock new markers
* Select different markers
* Read the instructions
* Check your record

### Ticket Links

#233

### Readiness Checklist
